core/lib/dal/src/blocks_dal.rs

```diff
@@ -1272,12 +1288,20 @@ impl BlocksDal<'_, '_> {
     pub async fn get_ready_for_execute_l1_batches(
```

Comment:
please comment what this method does (and what does the committed_tx_confirmed argument change)